src/init/readGraph/index.js

```diff
@@ -45,4 +46,9 @@ function readGraph (graph, pathToBundle, npmCachePath) {
   function getTarballName (name, version) {
     return name.replace(/\//g, '-') + '-' + version + '.tgz';
   }
+
+  function getPackageScope (name) {
+    var scope = name.substring(0, name.indexOf('/'));
+    return (scope !== name) ? scope : '';
+  }
```

Comment:
I'm not super happy with how we parse the scope of the package, so if you can see a better way let me know